content/03.results.md

```diff
@@ -211,17 +205,17 @@ To further validate the classifier's accuracy, we assessed _TP53_ scores for pat
 Indeed, we observed higher scores in 8/10 tumors from LFS patients (N = 8 patients)(**Table S3**).
 Although we observed low _TP53_ scores in two tumors from LFS patients (BS_DEHJF4C7 with a score of 0.09 and BS_ZD5HN296 with a score of 0.28), we confirmed from pathology reports that both patients were diagnosed with LFS and had a pathogenic germline variant in _TP53_.
 In addition, the tumor purity of these two LFS tumors was low (16% and 37%, respectively), suggesting the classifier may require a certain level of tumor purity to achieve good performance, as we expect _TP53_ to be intact in normal cells.
-These transcriptomic scores can be utilized to infer _TP53_ function in the absence of a predicted oncogenic _TP53_ alteration or DNA sequencing in general.
+We posit that these transcriptomic scores can be utilized to infer _TP53_ function in the absence of a predicted oncogenic _TP53_ alteration or DNA sequencing in general.
 
-We used gene expression data to predict telomerase activity using EXpression-based Telomerase ENzymatic activity Detection (`EXTEND`) [@doi:10.1038/s41467-020-20474-9] as a surrogate measure of malignant potential [@doi:10.1038/s41467-020-20474-9; @doi:10.1093/carcin/bgp268], such that higher `EXTEND` scores suggest increased malignant potential.
+We used gene expression data to predict telomerase activity using EXpression-based Telomerase ENzymatic activity Detection (`EXTEND`) [@doi:10.1038/s41467-020-20474-9] as a surrogate measure of malignant potential [@doi:10.1038/s41467-020-20474-9; @doi:10.1093/carcin/bgp268], such that higher `EXTEND` scores infer higher telomerase activity.
 While we did not find that tumors with _TERT_ promoter (TERTp) mutations (N = 6) had significantly higher telomerase activity scores than non-mutated tumors (Wilcoxon p-value = 0.1196), we observed that `EXTEND` scores significantly correlated with _TERC_ (R = 0.619, p < 0.01) and _TERT_ (R = 0.491, p < 0.01) expression (**Figure {@fig:S5}B-C**).
 Since catalytically-active telomerase requires a combination of full-length _TERT_, _TERC_, as well as accessory proteins [@url:https://pubmed.ncbi.nlm.nih.gov/9751630], we expect that `EXTEND` scores may not be exclusively correlated with _TERT_ alterations and expression.
```

Comment:
```suggestion
Since catalytically-active telomerase requires a combination of full-length _TERT_, _TERC_, and certain accessory proteins [@url:https://pubmed.ncbi.nlm.nih.gov/9751630], `EXTEND` scores may not be exclusively correlated with _TERT_ alterations and expression.
```

I feel like something is _missing here_? what else do we expect it's correlated with?

Reply:
Ah, this sentence is saying EXTEND scores are not solely due to TERT since telomerase is made up of many proteins

Reply:
how about this wording - changed above?